File: JPAGiggity/src/main/java/com/skilldistillery/giggity/entities/SellerReview.java
package com.skilldistillery.giggity.entities;

import java.time.LocalDateTime;

import javax.persistence.Column;

public class SellerReview {
	private int id;
	private String comment;
	private int rating;
	
	@Column(name="booking_id")
	private int bookingid;
	
	@Column(name="review_date")
	private LocalDateTime reviewDate;
}

File: ngGiggity/src/app/models/seller-review.ts
import { Booking } from './booking';
export class SellerReview {
  id: number;
  comment: string;
  rating: number;
  reviewDate: Date;
  booking: Booking;
  constructor(
    id?: number,
    rating?: number,
    reviewDate?: Date,
    comment?: string,
    booking?: Booking
  ) {
    this.id = id;
    this.comment = comment;
    this.rating = rating;
    this.reviewDate = reviewDate;
    this.booking = booking;
  }
}
